lessons/introduction-to-smartpy/06/06.mdx

```diff
@@ -6,18 +6,21 @@ editor:
   language: python
   startingCode: |
     import smartpy as sp
+
     class Plant(sp.Contract):
         def __init__(self, name):
-        		self.init(name = sp.string(name),
-            growth_rate = sp.int(10), health = sp.int(100))
-    
+
+            # Initilize the required attribute
+        		self.init(name = sp.string(name), attack = sp.int(10), defense = sp.int(10), growth_rate = sp.int(10), health = sp.int(100))
+
   answer: |  
     import smartpy as sp
+
     class Plant(sp.Contract):
         def __init__(self, name):
-        		self.init(name = sp.string(name),
-            growth_rate = sp.int(10), health = sp.int(100), is_alive = True)
-    
+
+            # Initilize the required attribute
+        		self.init(name = sp.string(name), attack = sp.int(10), defense = sp.int(10), growth_rate = sp.int(10), health = sp.int(100), is_alive = True)
```

Comment:
add line break here

Reply:
If we break the line, it will show error by the compiler.

Reply:
hmm, I think we have to re-create the complier, the principle being `focus on functionality and not the details of the implementation, cause we are getting that right off the bat by using Monaco diffing feature
I have to think about this, give me some time for this